wa.css

```diff
@@ -43,6 +43,11 @@ button#world-anvil img {
     Application Styling Rules
 --------------------------------------------- */
 
+/* World Overview */
+#world-anvil-browser .window-content {
+    max-height: 80vh;
+}
```

Comment:
Is this necessary?

Reply:
Without it, if you have a give world anvil world, it tends to take the whole screen.
I found it doesn't fit well, so added this.

What it does without this parameter : 
![image](https://user-images.githubusercontent.com/5261271/188272513-413d584a-3987-494c-9180-26cbca1dc5a0.png)